atos/modules/MQTTBridge/src/mqttsubscribercallback.cpp

```diff
@@ -0,0 +1,43 @@
+#include "mqttsubscribercallback.hpp"
+
+
+MQTTSubscriberCallback::MQTTSubscriberCallback(mqtt::async_client& client, mqtt::connect_options& connectionOptions, std::vector<std::string>& topics) :
+	client(client),
+	connectionOptions(connectionOptions),
+	topics(topics) {}
+
+
+MQTTSubscriberCallback::~MQTTSubscriberCallback() {}
+
+
+/**
+ * @brief Callback for when the client has connected, subscribe to the topics.
+ * 
+ * @param cause Cause.
+ */
+void MQTTSubscriberCallback::connected(const std::string& cause) {
```

Comment:
I think either something like "onConnectionEstablished" or simply "subscribe" would be a bit more clear. Although "subscribe" is used enough times already imo.

Reply:
Connected is an overriden function from mqtt::callback